archean/egg-archean.json

```diff
@@ -14,7 +14,7 @@
         "Ubuntu": "ghcr.io\/pelican-eggs\/yolks:ubuntu"
     },
     "file_denylist": [],
-    "startup": "echo \"$ADMINS\" > \"Archean-data\/server\/admins.txt\" && cd Archean-server-alpha && .\/Archean server",
+    "startup": "echo \"$ADMINS\" > \"Archean-data\/server\/admins.txt\", \/home\/container\/Archean server",
```

Comment:
Was the , a mistake or is it actually some operator I don't know of because it didn't work. Replacing it with && works fine.

Reply:
It should have been a ; not ,